docs/Token_Assumptions.md

```diff
@@ -0,0 +1,21 @@
+_Attestation Token use cases & assumptions_
```

Comment:
general comment: should we align to RATS glossary and call the attestation token "Evidence" instead?

Reply:
I think I'd prefer to leave it this way as I think it's more approachable for someone new to the area.